schema/sample/spectra/xrd.md

```diff
@@ -1,12 +1,21 @@
-- anode ("CuKa,CuKa2,CuKa1,CuKb1,MoKa,MoKa2,MoKa1,MoKb1,CrKa,CrKa2,CrKa1,CrKb1,FeKa,FeKa2,FeKa1,FeKb1,CoKa,CoKa1,CoKa2,CoKb1,AgKa,AgKa1,AgKa2,AgKb1")
-- environment (string): For example, in vacuo, N2
-- relativeHumidity (number, %)
-- remarks (html)
-- sampleHolder (string): For example, capillary
-- temperature (number, K)
 - xrd (array<object>): Powder x-ray diffraction:
+  - anode ("CuKa,CuKa2,CuKa1,CuKb1,MoKa,MoKa2,MoKa1,MoKb1,CrKa,CrKa2,CrKa1,CrKb1,FeKa,FeKa2,FeKa1,FeKb1,CoKa,CoKa1,CoKa2,CoKb1,AgKa,AgKa1,AgKa2,AgKb1")
+  - environment (string): For example, in vacuo, N2
   - instrument (object):
     - manufacturer (string)
     - model (string)
     - serialNumber (string)
     - software (string)
+  - jcamp (object):
+    - filename (string)
+  - relativeHumidity (number, %)
+  - remarks (html)
+  - sampleHolder (string): For example, capillary
+  - temperature (number, K)
+  - zone (array<object>): Used to store the results from peak picking
```

Comment:
A zone is something that has a from / to. I would rather go for peak like in IR

https://github.com/cheminfo/data_schema/blob/master/schema/sample/spectra/ir.md

Reply:
fixed it. Thanks!